homeassistant/components/climate/honeywell.py

```diff
@@ -182,8 +198,8 @@ def update(self):
                     data = val
 
         except StopIteration:
-            _LOGGER.error("Did not receive any temperature data from the "
-                          "evohomeclient API.")
+            _LOGGER.exception("Did not receive any temperature data from the "
```

Comment:
Good questions... I don't know... the code was here 6 month ago (https://github.com/home-assistant/home-assistant/commit/ada4de3ffb0476fc5f20a85bac3958637ec33552#diff-896a11ee424d43689886053ccedb92d9R178) maybe @turbokongen can help us with this ?